Tests/SnapshotTests/Utilities/SnapshotTestCase.swift

```diff
@@ -0,0 +1,11 @@
+import XCTest
+import Orbit
+import SwiftUI
+
+class SnapshotTestCase: XCTestCase {
+
+    override class func setUp() {
+        super.setUp()
+        Font.registerOrbitFonts()
```

Comment:
Can be removed

Reply:
I feel like we've had this discussion already 😄 

https://github.com/kiwicom/orbit-swiftui/pull/163#discussion_r890147043

Reply:
🤦